adal/src/androidTest/java/com/microsoft/aad/adal/DefaultTokenCacheStoreTests.java

```diff
@@ -29,6 +29,9 @@
 import android.support.test.InstrumentationRegistry;
 import android.support.test.runner.AndroidJUnit4;
 
+
```

Comment:
Nit: extra lines. We could use Ctrl + L in Android Studio to format the code automatically.